mapie/utils.py

```diff
@@ -154,23 +162,37 @@ def check_cv(
         If the cross-validator is not valid.
     """
     if cv is None:
-        return KFold(n_splits=5)
-    if isinstance(cv, int):
+        return KFold(
+            n_splits=5, shuffle=True, random_state=random_state
+        )
+    elif isinstance(cv, int):
         if cv == -1:
             return LeaveOneOut()
-        if cv >= 2:
-            return KFold(n_splits=cv)
-    if (
-        isinstance(cv, BaseCrossValidator)
-        or (cv == "prefit")
-        or (cv == "split")
-    ):
+        elif cv >= 2:
+            return KFold(
+                n_splits=cv, shuffle=True, random_state=random_state
+            )
+        else:
+            raise ValueError(
+                "Invalid cv argument. "
+                "Allowed integer values are -1 or int >= 2."
+            )
+    elif isinstance(cv, BaseCrossValidator):
         return cv
-    raise ValueError(
-        "Invalid cv argument. "
-        "Allowed values are None, -1, int >= 2, 'prefit', "
-        "or a BaseCrossValidator object (Kfold, LeaveOneOut)."
-    )
+    elif isinstance(cv, BaseShuffleSplit):
+        return cv
+    elif cv == "prefit":
+        return cv
+    elif cv == "split":
+        return ShuffleSplit(
+            n_splits=1, test_size=0.5, random_state=random_state
```

Comment:
Maybe it would be good to make the test_size a parameter of the function, and define the test_size as a class attribute, doing this, you able every one to change this ratio, it's a minor change but with a high added value

Reply:
I am not super fan to add a lot of parameters in class definition, but it makes sense. This change impacts also `MapieTimeSeriesRegressor`.

Reply:
Done.